Expensify/App
# Fix CreateMenu and mobile safari attachment picker

### Details
[This PR](https://github.com/Expensify/Expensify.cash/pull/2222) and [this PR](https://github.com/Expensify/Expensify.cash/pull/2424) made updates to an old version of `CreateMenu`, essentially missing the changes introduced to the component [here](https://github.com/Expensify/Expensify.cash/pull/2229/files).

Since we had two competing modules, `src/components/CreateMenu/` and `src/components/CreateMenu.js`, the latter was being used and the fixes to the attachment picker introduced [here](https://github.com/Expensify/Expensify.cash/pull/2229/files) were not taking effect.

This PR combines the histories of these two modules and creates a single `CreateMenu` component.

### Fixed Issues
Fixes (regression comment) https://github.com/Expensify/Expensify.cash/pull/2460#issuecomment-823604790

### Tests / QA Steps
1. Open the attachment picker.
2. Make sure you can choose an attachment and upload it.
3. Open the following menus, and make sure they appear in their correct locations:
    - Global create
    - Add attachment
    - Emoji menu
    - Edit photo (in Settings -> Profile)
    - ReportActionContextMenu

### Tested On


### Screenshots

#### Web
![image](https://user-images.githubusercontent.com/47436092/115605275-2df93400-a297-11eb-9ad1-15de0ea76455.png)

![image](https://user-images.githubusercontent.com/47436092/115605314-381b3280-a297-11eb-93cb-91ebe73dc980.png)

#### Mobile Web
![image](https://user-images.githubusercontent.com/47436092/115605362-4701e500-a297-11eb-8ff7-b315851b05b8.png)

![image](https://user-images.githubusercontent.com/47436092/115605420-57b25b00-a297-11eb-94a7-8e101908c544.png)

#### Desktop


#### iOS


#### Android
